plugins/keycloak-backend/__fixtures__/helpers.ts

```diff
@@ -2,14 +2,19 @@ import { getVoidLogger } from '@backstage/backend-common';
 import { TaskInvocationDefinition, TaskRunner } from '@backstage/backend-tasks';
 import { EntityProviderConnection } from '@backstage/plugin-catalog-node';
 
-import { groupMembers, groups, users } from './data';
+import {
+  groupMembers,
+  topLevelGroups23orHigher,
+  topLevelGroupsLowerThan23,
+  users,
+} from './data';
 
 export const BASIC_VALID_CONFIG = {
   catalog: {
     providers: {
       keycloakOrg: {
         default: {
-          baseUrl: 'http://localhost:8080/auth',
+          baseUrl: 'http://localhost:8080',
```

Comment:
I was wondering why `auth` was removed and saw that there's a comment in our [README](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/keycloak-backend/README.md#new-backend-configuration) stating that it's not needed for Keycloak v17+.  Can you update the readme examples (on both the legacy and new backends) to not have `auth` so it's correct by default? 

I also checked our [product docs](https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/configuring_plugins_in_red_hat_developer_hub/rhdh-installing-dynamic-plugins#advanced-configuration) and we mention it there as well.  Let's make sure this gets updated for 1.4 as well

Reply:
Updated! I'll also create a separate issue to update this in our products.